commands/account/list.ts

```diff
@@ -112,12 +109,22 @@ export async function handler(
     ])
   );
 
+  let defaultAccount;
+
+  try {
+    defaultAccount = getConfigDefaultAccount();
+  } catch (e) {}
+
   logger.log(i18n(`${i18nKey}.configPath`, { configPath: configPath! }));
-  logger.log(
-    i18n(`${i18nKey}.defaultAccount`, {
-      account: getConfigDefaultAccount()!,
-    })
-  );
+
+  if (defaultAccount) {
+    logger.log(
+      i18n(`${i18nKey}.defaultAccount`, {
+        account: defaultAccount.name,
```

Comment:
Assuming that in the new `getConfigDefaultAccount()` function you've written that `name` does exist, correct?